src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets

```diff
@@ -445,6 +446,52 @@ Copyright (c) .NET Foundation. All rights reserved.
+    </PropertyGroup>
+
+    <GetAssemblyInfo RelativeFilePath="$(_CoreGenerateSatelliteAssembliesTemplateFile)" >
+      <Output TaskParameter="CompanyName" PropertyName="_CompanyNameFromTemplateFile" />
+      <Output TaskParameter="LegalCopyright" PropertyName="_LegalCopyrightFromTemplateFile" />
+      <Output TaskParameter="Comments" PropertyName="_CommentsFromTemplateFile" />
+      <Output TaskParameter="FileVersion" PropertyName="_FileVersionFromTemplateFile" />
+      <Output TaskParameter="ProductVersion" PropertyName="_ProductVersionFromTemplateFile" />
+      <Output TaskParameter="ProductName" PropertyName="_ProductNameFromTemplateFile" />
+      <Output TaskParameter="FileDescription" PropertyName="_FileDescriptionFromTemplateFile" />
+      <Output TaskParameter="AssemblyVersion" PropertyName="_AssemblyVersionFromTemplateFile" />
+    </GetAssemblyInfo>
+
+    <ItemGroup>
+      <SatelliteAssemblyAttribute Include="System.Reflection.AssemblyCompanyAttribute" Condition="'$(_CompanyNameFromTemplateFile)' != ''">
+        <_Parameter1>$(_CompanyNameFromTemplateFile)</_Parameter1>
+      </SatelliteAssemblyAttribute>
+      <SatelliteAssemblyAttribute Include="System.Reflection.AssemblyCopyrightAttribute" Condition="'$(_LegalCopyrightFromTemplateFile)' != ''">
+        <_Parameter1>$(_LegalCopyrightFromTemplateFile)</_Parameter1>
+      </SatelliteAssemblyAttribute>
+      <SatelliteAssemblyAttribute Include="System.Reflection.AssemblyDescriptionAttribute" Condition="'$(_CommentsFromTemplateFile)' != ''">
+        <_Parameter1>$(_CommentsFromTemplateFile)</_Parameter1>
+      </SatelliteAssemblyAttribute>
+      <SatelliteAssemblyAttribute Include="System.Reflection.AssemblyFileVersionAttribute" Condition="'$(_FileVersionFromTemplateFile)' != ''">
+        <_Parameter1>$(_FileVersionFromTemplateFile)</_Parameter1>
+      </SatelliteAssemblyAttribute>
+      <SatelliteAssemblyAttribute Include="System.Reflection.AssemblyInformationalVersionAttribute" Condition="'$(_ProductVersionFromTemplateFile)' != ''">
+        <_Parameter1>$(_ProductVersionFromTemplateFile)</_Parameter1>
+      </SatelliteAssemblyAttribute>
+      <SatelliteAssemblyAttribute Include="System.Reflection.AssemblyProductAttribute" Condition="'$(_ProductNameFromTemplateFile)' != ''">
+        <_Parameter1>$(_ProductNameFromTemplateFile)</_Parameter1>
+      </SatelliteAssemblyAttribute>
+      <SatelliteAssemblyAttribute Include="System.Reflection.AssemblyTitleAttribute" Condition="'$(_FileDescriptionFromTemplateFile)' != ''">
+        <_Parameter1>$(_FileDescriptionFromTemplateFile)</_Parameter1>
+      </SatelliteAssemblyAttribute>
+      <SatelliteAssemblyAttribute Include="System.Reflection.AssemblyVersionAttribute" Condition="'$(_AssemblyVersionFromTemplateFile)' != ''">
+        <_Parameter1>$(_AssemblyVersionFromTemplateFile)</_Parameter1>
+      </SatelliteAssemblyAttribute>
```

Comment:
I would consider having the task return these items and renaming it to GetAssemblyAttributes. It will be more efficient with less code in XML to maintain.